src/main/webapp/WEB-INF/spring/appServlet/servlet-context.xml

```diff
@@ -89,7 +89,12 @@
     <beans:bean class="org.ambraproject.rhino.rest.LoggingInterceptor"/>
   </mvc:interceptors>
 
+  <beans:bean id="flyway" class="org.flywaydb.core.Flyway" init-method="migrate">
+    <beans:property name="locations" value="classpath:migrations/" />
```

Comment:
This value defaults to  "classpath:db/migration". Since you're moving the migration files anyways, you might as well just move them to the conventional location.